sympy/matrices/matrices.py

```diff
@@ -340,9 +340,9 @@ def inv(self, method=None, **kwargs):
         Inverse is found using Gaussian elimination & LU, with former being default.
 
         Raises
-        ========
-
-        If the determinant of the matrix is zero, a ``ValueError`` is raised.
+        ------
+        ValueError
+        If the determinant of the matrix is zero.
```

Comment:
@jksuom Does it Look Good ?

I removed some lines that were there in the ``def _eval_inverse`` ( ``dense.py``) as ``try_block_diag`` and ``iszerosfunc`` are not there in the code under ``inv`` ( ``matrices.py``).